src/components/Contexts/StepperDialogProvider.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext, useState } from 'react'
+
+const StepperDialogContext = createContext()
+
+const StepperDialogProvider = ({ children }) => {
+  const [isStepperDialogOpen, setIsStepperDialogOpen] = useState(false)
+  const [dialogStepperTitle, setStepperDialogTitle] = useState('')
+  const [currentPages, setCurrentPages] = useState([])
+
+  const dialogStepper = {
+    isStepperDialogOpen,
+    setIsStepperDialogOpen,
+    currentPages,
+    setCurrentPages,
+    dialogStepperTitle,
```

Comment:
Shouldn't it be stepperDialogTitle?

Also there are some other `dialogStepper` in other files.